### PR TITLE
Sample added for Cloud Platform

### PR DESCRIPTION
**Pull request by Venkateshmuruganandam**
Hi All,
Sample added for following platform.

Regards
Venkatesh

### CODE REVIEW

File: Getting-Started/AWS/AWS_Lambda/Create-Word-Document/aws-lambda-tools-defaults.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "configuration" : "Release",
+    "function-architecture" : "x86_64",
+    "function-runtime"      : "dotnet6",
+    "function-memory-size"  : 256,
+    "function-timeout"      : 30,
+    "function-handler"      : "Create-Word-Document::Create_Word_Document.Function::FunctionHandler",
+    "framework"             : "net6.0",
+    "function-name"         : "Create-Word",
+    "package-type"          : "Zip",
+    "function-role"         : "arn:aws:iam::142887710098:role/LambdaRole",
```
Comment (MohanaselvamJothi) [inline Getting-Started/AWS/AWS_Lambda/Create-Word-Document/aws-lambda-tools-defaults.json:20]: change this

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: Getting-Started/AWS/Console_Application/Create-Word-Document/Program.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+using System.Text;
+using System.Threading.Tasks;
+using Amazon;
+
+namespace Create_Word_Document
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
```
[the remark this answers is not in the document]
Comment (Venkateshmuruganandam) [reply Getting-Started/AWS/Console_Application/Create-Word-Document/Program.cs:17]: updated

File: Getting-Started/Azure/Azure_App_Service/Create-Word-Document/Controllers/HomeController.cs [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+                textRange = paragraph.AppendText("Price: $3,578.27\r") as WTextRange;
+                textRange.CharacterFormat.FontSize = 12f;
+                textRange.CharacterFormat.FontName = "Times New Roman";
+                //Appends paragraph.
+                section.AddParagraph();
+
+                //Saves the Word document to MemoryStream.
+                MemoryStream stream = new MemoryStream();
+                document.Save(stream, FormatType.Docx);
+                stream.Position = 0;
```
[the remark this answers is not in the document]
Comment (Venkateshmuruganandam) [reply Getting-Started/Azure/Azure_App_Service/Create-Word-Document/Controllers/HomeController.cs]: updated

File: Getting-Started/AWS/AWS_Lambda/Create-Word-Document/aws-lambda-tools-defaults.json (hunk 1 above)
Comment (Venkateshmuruganandam) [reply Getting-Started/AWS/AWS_Lambda/Create-Word-Document/aws-lambda-tools-defaults.json]: updated.

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**